test/e2e_node/docker_util.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e_node
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/blang/semver"
+	"github.com/docker/docker/client"
+)
+
+// getDockerAPIVersion returns the Docker's API version.
+func getDockerAPIVersion() (semver.Version, error) {
+	c, err := client.NewClient("unix:///var/run/docker.sock", "", nil, nil)
```

Comment:
security_context.go gets the version using `docker -v`, which is different than how we get the version in docker_util.go. I guess we can probably change it in a separate PR?